app/helpers/bookings_helper.rb

```diff
@@ -1,7 +1,74 @@
 module BookingsHelper
 
-  def fetch_booking_info
+  def update_saved(booking_params)
+    if @booking.update(booking_params)
+        @booking.save
+        message = "Update successful."
+    else
+      message = "Update failed."
+    end
+  end
+
+  def booking_params
+    params.require(:booking).
+    permit( :no_of_passengers,
+            :flight_id,
+            :confirmation_code,
+            :customer_id,
+            :id,
+            passengers_attributes:
+                                [:name,
+                                 :email,
+                                 :_destroy
+                                ] )
+  end
+
+  def flight_reseter(column, id, num)
+    booking = Booking.new
+    booking_info = booking.get_all_bookings(column, id)
+    {
+      code: booking_info[0]["flight_code"],
+      dest: booking_info[0]["destination"],
+      dept: booking_info[0]["departure"],
+      cost: booking_info[0]["cost"],
+      airline: booking_info[0]["airline"],
+      flight_datetime: booking_info[0]["dept_date"],
+      pass: num,
+      booking_id: booking_info[0]["id"]
+    }
+  end
+
+  def change_no_of_pass(passenger_num, id)
```

Comment:
Unused method argument - `passenger_num`. If it's necessary, use `_` or `_passenger_num` as an argument name to indicate that it won't be used.